.circleci/config.yml

```
@@ -43,6 +44,10 @@ jobs:
             echo "Website added to IPFS: https://ipfs.io/ipfs/$hash"
 
             if [ "$CIRCLE_BRANCH" == "master" ] ; then
+              dnslink-dnsimple -d $DEV_DOMAIN -r _dnslink -l /ipfs/$hash
```

```suggestion
              dnslink-dnsimple -d $DOMAIN -r _dnslink.dev -l /ipfs/$hash
```
The record name includes the .dev rather than the zone.